lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -65,23 +65,12 @@ class ReportRenderer {
         Util.formatDateTime(report.fetchTime);
     this._dom.find('.lh-product-info__version', header).textContent = report.lighthouseVersion;
     const url = /** @type {HTMLAnchorElement} */ (this._dom.find('.lh-metadata__url', header));
-    url.href = report.finalUrl;
-    url.textContent = report.finalUrl;
     const toolbarUrl = /** @type {HTMLAnchorElement}*/ (this._dom.find('.lh-toolbar__url', header));
-    toolbarUrl.href = report.finalUrl;
-    toolbarUrl.textContent = report.finalUrl;
-
-    this._dom.find('.lh-env__item__ua', header).textContent = report.userAgent;
-
-    const env = this._dom.find('.lh-env__items', header);
-    const environment = Util.getEnvironmentDisplayValues(report.configSettings || {});
-    environment.forEach(runtime => {
-      const item = this._dom.cloneTemplate('#tmpl-lh-env__items', env);
-      this._dom.find('.lh-env__name', item).textContent = runtime.name;
-      this._dom.find('.lh-env__description', item).textContent = runtime.description;
-      env.appendChild(item);
-    });
+    url.href = url.textContent = toolbarUrl.href = toolbarUrl.textContent = report.finalUrl;
 
+    const emuDescs = Util.getEmulationDescriptions(report.configSettings || {});
```

Comment:
the `lation` is just too much, huh?

let's use emulation and nix the `/** @type HTMLAnchorElement`